src/SourceBuild/tarball/content/build.sh

```diff
@@ -5,11 +5,12 @@ IFS=$'\n\t'
 usage() {
     echo "usage: $0 [options]"
     echo "options:"
+    echo "  --clean-while-building             cleans each repo after building (reduces disk space usage)"
+    echo "  --poison                           build with poisoning checks"
     echo "  --online                           build using online sources"
```

Comment:
```suggestion
    echo "  --online                           build using online sources"
    echo "  --poison                           build with poisoning checks"
```

Alphabetization is hard  :)

Reply:
To be fair, I only said they should be alphabetized...lol